js/map_utils.js

```diff
@@ -242,18 +242,39 @@ function documentCategorySwitch(category, prevCategory, action) {
 
 var toolTipTimer = null;
 var toolTipDelay = 1000; //ms
-function showToolTip(d, category) {
 
-  // change tooltip
-  d3.select(".tooltip")
-    .classed("hidden", false)
-    .style("left", (d3.event.pageX + 35) + "px")
-    .style("top", (d3.event.pageY - 50) + "px");
-  d3.select(".tooltip")
-    .html(d.COUNTY + ", " + d.STATE_ABBV + "<br/>" + 
-            "Population: " + d.countypop + "<br/>" +
-            categoryToName(category) + ": " + 
-              d[[category]] + " " + "MGD");
+function updateLegendText(d, category) {
+
+  waterUseViz.elements
+    .buttonBox
+    .selectAll("#legend-title")
+    .text(d.COUNTY + ", " + d.STATE_ABBV);
+
+  waterUseViz.elements
+    .buttonBox
+    .selectAll("#total_button_text")
+    .text(d[["total"]]);
+    
+  waterUseViz.elements
+    .buttonBox
+    .selectAll("#publicsupply_button_text")
+    .text(d[["publicsupply"]]);
+    
+  waterUseViz.elements
+    .buttonBox
+    .selectAll("#irrigation_button_text")
+    .text(d[["irrigation"]]);
+    
+  waterUseViz.elements
+    .buttonBox
+    .selectAll("#industrial_button_text")
+    .text(d[["industrial"]]);
+
+  waterUseViz.elements
+    .buttonBox
+    .selectAll("#thermoelectric_button_text")
+    .text(d[["thermoelectric"]]);
+
```

Comment:
Getting fancy with the selectors! Do they all need to be edited every time? What if we just update the two text elements that are visible? Also, note that there's a `select()` function at your disposal when you just want to select one thing.
```
waerUseViz.elements.buttonBox
  .select('#legend-title')
  .text(d.COUNTY + ", " + d.STATE_ABBV);

waterUseViz.elements.buttonBox
  .select("#" + category + "-button-text")
  .text(d[[category]]);
```

Reply:
Hmm...I had done it like you described, but then when you hovered over the selectors, there were old numbers that were showing up...but THEN I realized no matter what we need the `clearLegendText`..so yeah, maybe we could go back to this scheme...let me try